packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -52,9 +53,13 @@ export const start = () => {
     // this to set up, eg. Redux stores.
     const locals = {}
 
+    if (window.__CONFIG__) {
+        setConfig(window.__CONFIG__)
+    }
+
     // AppConfig.restore *must* come before getRoutes()
     AppConfig.restore(locals, window.__PRELOADED_STATE__.__STATE_MANAGEMENT_LIBRARY)
-    const routes = getRoutes(locals)
+    const routes = getRoutes(locals, window.__CONFIG__)
```

Comment:
This doesn't match the signature of `getRoutes` https://github.com/SalesforceCommerceCloud/pwa-kit/pull/421/files#diff-bb1f55b8c21cbabdb10e9a153bef5868e2c17ff9a1a46e658bc01a59b84f3002R412

Reply:
Good call.. I think this might have been an artifact of a previous implementation. Good eye